site/content/en/docs/Getting Started/edit-first-gameserver-go.md

```diff
@@ -103,7 +96,7 @@ kubectl describe gameserver
 Let's retrieve the IP address and the allocated port of your Game Server:
 
 ```bash
-kubectl get gs simple-game-server -o jsonpath='{.status.address}:{.status.ports[0].port}'
+kubectl get gameservers | grep Ready | awk '{print $3":"$4 }'
```

Comment:
Did the command that was here before not work for you?

Reply:
No, since the generatedName is different, let's say this is the game server that was created `simple-game-server-kmpw5 ` so you would run into the issue of `Error from server (NotFound): gameservers.agones.dev "simple-game-server" not found` when running the above command. I can also leave the original command and add a note on it saying that you should use the generated name of the game server...

Reply:
Why not just remove the name but leave the `jsonpath` extraction? (It's more universal, though `awk` is universal at this point as well.)

Reply:
okok perfect, i've updated the command